packages/config/src/projects/arbitrum.ts

```diff
@@ -131,6 +131,7 @@ export const arbitrum: Project = {
               href: 'https://developer.offchainlabs.com/docs/inside_arbitrum#rules-for-confirming-or-rejecting-rollup-blocks',
             },
           ],
+          risks: [EXITS.RISK_CENTRALIZED_BLOCK_PRODUCER],
```

Comment:
We definitely need a reference here.

Reply:
https://offchain.medium.com/mainnet-for-everyone-27ce0f67c85e